doc/double_deposit.md

```diff
@@ -0,0 +1,69 @@
+Similar to ICA, Sending an ibc packet from chain A, include signature of chainB user's signature and execute Tx on chain B according to the signature verification result.
+
+a tx structure will similar to this:
+
+```go
+{
+     senders: []string{"Achain1xxxxxxxxx", "Bchain1xxxxx"},
+     tokens: []*sdk.Coin{{Denom: "aside", Amount: 1000},{Denom: "bside", Amount: 1000}},
+     signature: []byte
+}
+```
```

Comment:
It would be better to wrap a sub tx which is executed on counterpart chain.

Reply:
that's just main transaction. MsgDeoubleDeposit. 
it already include subTx information. 
for example, sender, amount for tuning bank types.MsgSend, that's just I'm not sending subTx additionally. 
we can config transaction with current information without subTx raw data.